[DO NOT MERGE] Enable creating add-in in one step in a new tab (#4314)
OfficeDev/office-js-docs-pr: docs/quickstarts/word-quickstart-experiment.md

---

# Build your first Word task pane add-in

In this article, you'll walk through the process of building a Word task pane add-in.

## Create the add-in

[!include[Choose your editor](../includes/quickstart-choose-editor-experiment.md)]

# [One-line script](#tab/script)

### Prerequisites

You must have Office connected to a Microsoft 365 subscription.

> [!NOTE]
> If you don't already have Office, you can [join the Microsoft 365 developer program](https://developer.microsoft.com/microsoft-365/dev-program) to get a free, 90-day renewable Microsoft 365 subscription to use during development.

### Create the add-in project

You can run the one-line script on Windows or on macOS. It performs the following to set up the add-in development environment.

On Windows:
- Installs Node.js, and Git, if they aren't found on your machine.
- Downloads the files of the sample add-in project.
- Opens the add-in project in Visual Studio Code, if installed. If Visual Studio Code isn't installed on your machine, the folder that contains the add-in project files opens instead.
- Prompts you to run the add-in.

On macOS:
- Downloads the files of the sample add-in project.

> - Opens the folder that contains the add-in project files. Or opens the add-in project in Visual Studio Code if it is installed and configured to support launching from terminal. 

Reviewer: ```suggestion
- Opens the folder that contains the add-in project files. If Visual Studio Code is installed and configured to launch projects from the terminal, the add-in project opens there instead.
```

- Prompts you to run the add-in.

> > [!NOTE]

Reviewer: ```suggestion

> [!NOTE]
```

> In addition to Windows and macOS, we're working to support the script on other platforms.

If you're using Windows, run the following from a command prompt.
```command&nbsp;line
iwr aka.ms/wordaddin/helloworld -outfile wordhelloworld.bat; saps wordhelloworld.bat
```
If you're using macOS, run the following in a terminal.
```command&nbsp;line
bash <(curl -L -s aka.ms/wordaddin/helloworld_mac)
```

### Try it out

Once your add-in project is set up, it automatically opens in Visual Studio Code. If you don't have Visual Studio Code installed on your machine, the folder containing the files of your add-in project opens instead.

> If you choose to run the add-in when prompted, a Word document opens, displaying the add-in task pane. choose the **Run** link to add the text "Hello World" to the document in blue font. 

Reviewer: ```suggestion
If you choose to run the add-in when prompted, a Word document opens, displaying the add-in task pane. From the task pane, select **Run** to add the text "Hello World" to the document in blue font. 
```
